boards/adafruit/feather_nrf52840/Kconfig.defconfig

```diff
@@ -9,4 +9,6 @@ if BOARD_ADAFRUIT_FEATHER_NRF52840
 config BT_CTLR
 	default BT
 
+source "${ZEPHYR_BASE}/boards/common/usb/Kconfig.cdc_acm_serial"
```

Comment:
This would enable CDC ACM on the 2 non UF2 variants of this board which currently don't have it enabled. Is that the intended outcome?

I struggled to deal with this when creating these variants. I've since realised perhaps using if BOARD_TARGET =  would have been a better way of dealing with some of the variations. Perhaps that would work here. It would be great to be using common definitions.

Reply:
fixed

Reply:
Since you have only selected this for the specific boards in Kconfig you don’t need the if here do you?

Reply:
Correct, it can now be removed. Thanks.